pkg/util/external/rdt/manager.go

```diff
@@ -14,15 +14,6 @@
 
 package rdt
 
-import (
-	"errors"
-)
-
-const (
-	// SubManagerName is the name of NetworkManager.
-	SubManagerName = "rdt"
-)
-
 // RDTManager provides methods that control RDT related resources.
```

Comment:
add comments here to clarify that CRI has already supports rdt related functionality, but containerd is lack of this, so we have to implement here now, but we plan to switch to the standard CRI interface in the future

Reply:
cc @csfldf

Reply:
done